pywr-schema/src/metric_sets/mod.rs

```diff
@@ -91,18 +131,46 @@ pub struct MetricSet {
     pub name: String,
     pub metrics: Vec<Metric>,
     pub aggregator: Option<MetricAggregator>,
+    #[serde(default)]
+    filters: MetricSetFilters,
```

Comment:
This should be `pub` as it is part of the schema. It will require making `MetricSetFilters` public also.

We should also add to the docstring that these filters append to `metrics`, but will avoid creating duplicates. 

Should the `metrics` field be `Option<_>` now?